psutil/arch/freebsd/specific.c

```diff
@@ -517,7 +517,7 @@ psutil_swap_mem(PyObject *self, PyObject *args) {
 }
 
 
-#if defined(__FreeBSD_version) && __FreeBSD_version >= 800000
+#if defined(__FreeBSD_version) && __FreeBSD_version >= 701000
```

Comment:
Does this mean FreeBSD 7 was not supported and now it is?

Reply:
Yes, I am able to compile on FreeBSD 7.1 after the change in `4b269a7`:

```shell
> python2 -c 'import os, psutil; print os.uname()[3], psutil.Process().environ()["OSTYPE"]'
FreeBSD 7.1-RELEASE #0: Thu Jan  1 08:58:24 UTC 2009     root@driscoll.cse.buffalo.edu:/usr/obj/usr/src/sys/GENERIC 
FreeBSD
```

Actually, we have bad rain here the whole day ;-),  and just for fun I've even managed to:
```shell
> python2 -c 'import os, psutil; print os.uname()[3], "\n", psutil.Process().environ()["OSTYPE"]'
FreeBSD 6.4-RELEASE #0: Wed Nov 26 08:21:48 UTC 2008     root@palmer.cse.buffalo.edu:/usr/obj/usr/src/sys/GENERIC 
FreeBSD
```
... though for compiling on `6.4`, I had to cut off some source bits in
`psutil/arch/freebsd/proc_socks.c`, `psutil/arch/freebsd/specific.c` and `psutil/arch/freebsd/sys_socks.c` because the 6.4 kernel interface is too old for that code...